LanguageFeatures/Extension-types/static_analysis_extension_types_A33_t03.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion An extension type V is always-exhaustive if and only if its
+/// instantiated representation type is always-exhaustive.
+///
+/// @description Check that an extension type is not always-exhaustive if its
+/// representation type is not always-exhaustive but adding an exhaustive case
+/// removes a compile-time error
```

Comment:
I don't think it's possible to match up a test like this one with the concept of an always-exhaustive type: Such types are taken into account during the static analysis of switch statements (a switch statement must be exhaustive if its scrutinee has a static type which is always-exhaustive), but they simply aren't relevant to switch expressions (they must _always_ be exhaustive, period). We should probably just drop this test.

Reply:
Ok, dropped